src/docs/release-notes/R25/Summary.mdx

```diff
@@ -0,0 +1 @@
+Migrated all our [Angular](https://brightlayer-ui-components.github.io/angular) and [React](https://brightlayer-ui-components.github.io/react) component developer documentation from Storybook to a new site with more examples and more detailed usage explanations; added support for React Native v0.69; added [icon style guide](/style/icon-guidelines).
```

Comment:
See my first comment, I would restructure this sentence just a little bit to put more emphasis on what's new vs storybook.